GKI3dNavPlanner:
  environment_type: XYT_3dCollisions
  planner_type: ARAPlanner
  allocated_time: 10.0
  initial_epsilon: 5.0
  forward_search: true

  scene_update_name: /get_planning_scene
  allowed_collision_links:
    - br_caster_r_wheel_link
    - br_caster_l_wheel_link
    - bl_caster_r_wheel_link
    - bl_caster_l_wheel_link
    - fr_caster_r_wheel_link
    - fr_caster_l_wheel_link
    - fl_caster_r_wheel_link
    - fl_caster_l_wheel_link

